index.html

```diff
@@ -12,10 +12,9 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta name="generator" content="PWA Starter Kit">
-    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
-
     <title>My app</title>
+    <meta name="generator" content="PWA Starter Kit">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
Do you mind adding a space between the key-value pairs for readability, e.g. `"width=device-width, initial-scale=1"`? Otherwise LGTM.

Reply:
It feels like a wasted byte (although it probably isn’t given gzip/brotli), but whatever, done :P